PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,4 +8,6 @@
 
 - Has this change been tested? If so, mention the version number that the test was based on.
 
+- So we have a better idea of who you are Discord Handle?
```

Comment:
I'd suggest turning it into a proper question, "... of who you are, what is your Discord Handle?"